### DISCUSSION TITLE
can i fly UMX pitts s 1-s with my jumper t-lite 1 what protocol do i use ?

### DISCUSSION THREAD

**Discussion by Pappaburn123** [EdgeTX/edgetx#4639] [General]
can i fly UMX pitts s 1-s with my jumper t-lite 1 what protocol do i use ?

**Comment by frankiearzu** [comment]
Yes if you have the 4-in-1 MultiModule,   Protocol DSM-Auto..   If that does not bind, try DSM-X2F.

**Reply by Pappaburn123** [reply]
Thank you for this :)
